app/images/spinnerlight.svg

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm adding this file to git because we lost the original one years ago (and might use this instead of other png spinners in the future)